modal/volume.py

```diff
@@ -124,7 +148,50 @@ def f():
         ```
 
         """
-        return _Volume.new()._persist(label, namespace, environment_name)
+        return _Volume.from_name(label, namespace, environment_name, create_if_missing=True)
+
+    @staticmethod
+    async def lookup(
+        label: str,
+        namespace=api_pb2.DEPLOYMENT_NAMESPACE_WORKSPACE,
+        client: Optional[_Client] = None,
+        environment_name: Optional[str] = None,
+        create_if_missing: bool = False,
+    ) -> "_Volume":
+        """Lookup a network file system with a given name and tag.
```

Comment:
Nit: "network file system"

Also I don't understand "given name *and tag*"; the example only uses a single argument.

Reply:
sorry, this is copy paste two generations – let me fix it and also the other place